src/scripts/ui/inline-roll-links.ts

```diff
@@ -210,9 +216,18 @@ export const InlineRollLinks = {
             return;
         }
 
-        const flavor = target.attributes.getNamedItem("data-pf2-repost-flavor")?.value ?? "";
-        const showDC =
-            target.attributes.getNamedItem("data-pf2-show-dc")?.value ?? (document?.hasPlayerOwner ? "all" : "gm");
+        const defaultVisibility = document?.hasPlayerOwner ? "all" : "gm";
+        let content;
```

Comment:
In this project we're kinda highly alergic to let. Use an IIFE if you need to.

Also lower down I think you don't need to check against explicitly undefined, checking if its truthy should be enough.

Reply:
Sounds good, I've refactored the PR to remove `let` bindings. The comparison to undefined used here is also to check between an empty string and `undefined`, so switching it to a simple truthiness check breaks the logic.